src/pilot/assign_role.py

```diff
@@ -428,16 +457,17 @@ def define_storage_logical_disks(drac_client, raid_controller_name):
     #
```

Comment:
remaining_physical_disks would change to be a clone of the raid_cntlr_physical_disks dict, and the above code would update the list of physical disks for the used RAID controller to not include the disks that we used in the RAID1.